src/bot.js

```diff
@@ -75,6 +99,27 @@ try {
                         }
                     });
                     break;
+                case '!mygames':
+                    userName = message.author.username;
+                    // Game list will be everything after the space 
+                    gameStr = message.content.toLocaleLowerCase();
+                    if(gameStr.indexOf(" ") != -1){
+                      gameStr = gameStr.substring(gameStr.indexOf(" ")+1, gameStr.length);
```

Comment:
```suggestion
                      gameStr = gameStr.substring(gameStr.indexOf(" ")+1);
```
the to parameter isn't needed if you're substring ends before the end